client/rhel/spacewalk-client-tools/src/up2date_client/rhnreg_constants.py

```diff
@@ -193,62 +191,62 @@
-"Contact your Satellite administrator about this problem. Once you make the "
+"Contact your Uyuni administrator about this problem. Once you make the "
 "appropriate active subscriptions available in your account, you may browse "
-"to this system's profile in the RHN web interface and subscribe this system "
+"to this system's profile in the Uyuni web interface and subscribe this system "
 "to software channels via the software > software channels tab.")
 SLOTS_TITLE             = _("Service Level:")
-OK_SLOTS                = _("Depending on what Red Hat Satellite modules are associated with a system, you'll "
+OK_SLOTS                = _("Depending on what Uyuni system types are associated with a system, you'll "
                             "enjoy different benefits. The following are the "
-                            "Red Hat Satellite modules associated with this system:")
+                            "Uyuni system types associated with this system:")
 SLOTS                   =  SLOTS_TITLE + "\n" + OK_SLOTS + "\n%s"
 FAILED_SLOTS            = _("This system was unable to be associated with the "
-                            "following RHN module(s) because there were "
+                            "following Uyuni system type(s) because there were "
                             "insufficient subscriptions available in your account:")
-MANAGEMENT              = _("Management module: automatic updates, systems "
+MANAGEMENT              = _("Management type: automatic updates, systems "
                             "grouping, systems permissions, system package profiling, "
                             "bare-metal provisioning, existing state provisioning, "
                             "rollbacks, configuration management")
 
-VIRT = _("Virtualization module: software updates for a limited number of "
+VIRT = _("Virtualization type: software updates for a limited number of "
         "virtual guests on this system.")
 
 
 VIRT_FAILED = _("<b>Warning:</b> Any guest systems you create on this system "
-        "and register to RHN will consume Red Hat Enterprise Linux "
+        "and register to Uyuni will consume system "
         "subscriptions beyond this host system's subscription. You will need "
         "to: (1) make a virtualization system entitlement available and "
         "(2) apply that system entitlement to this "
-        "system in RHN's web interface if you do not want virtual guests of "
+        "system in Uyuni web interface if you do not want virtual guests of "
         "this system to consume additional subscriptions.")
 
 NO_SYS_ENTITLEMENT         = _("This system was unable to be associated with "
-"any RHN service level modules. This system will not receive any software "
-"updates to keep it secure and supported. Contace your Satellite administrator "
+"any Uyuni system types. This system will not receive any software "
+"updates to keep it secure and supported. Contace your Uyuni administrator "
```

Comment:
```suggestion
"updates to keep it secure and supported. Contact your Uyuni administrator "
```

I skipped a lot of the above changes but same comments as before:
a) I wonder if we really run this code these days
b) If we do, we should have a PRODUCT_NAME replacement with either "Uyuni" or "SUSE Manager"